backend/audit/validators.py

```diff
@@ -534,9 +534,15 @@ def validate_pdf_file_integrity(file):
                 "We were unable to process the file you uploaded because it is encrypted."
             )
 
-        all_text = "".join([p.extract_text() for p in reader.pages])
-
-        if len(all_text) == 0:
+        text_length = 0
+        for page in reader.pages:
+            page_text = page.extract_text()
+            text_length += len(page_text)
+            # If we find 5000 characters, we're content.
+            if text_length > 5000:
```

Comment:
if in the end we're really only concerned about the binary question of whether or not the PDF contains _any_ readable text, can we remove this arbitrary 5000 character upper bound in favor of bailing out as soon as we find a page where `text_length > 0`?

Shouldn't functionally change anything, but would reduce some obfuscation -- why 5000? why not 1,000 or 10,000?